src/backend/recorders/baseRecorder.ts

```diff
@@ -131,6 +134,22 @@ export abstract class BaseRecorder<T extends WebGLObject> implements IRecorder {
     public appendRecordedInformation(capture: ICapture): void {
         capture.frameMemory[this.objectName] = this.frameMemory;
         capture.memory[this.objectName] = this.memoryPerSecond;
+        capture.objects[this.objectName] = {};
+        for (const id in this.objects) {
+            if (this.objects.hasOwnProperty(id)) {
+                const object = this.objects[id].deref();
+                if (object) {
+                    const source = WebGlObjects.getWebGlObjectSource(object);
+                    if (source) {
+                        capture.objects[this.objectName][id] = source;
+                    } else {
+                        console.log("Object without source");
+                    }
+                } else {
+                    console.log("Object which was already free'd");
```

Comment:
I need a way to display this properly; I never saw these messages, and I assume they got compiled away?